### ISSUE TITLE
Exclude files?

### ISSUE THREAD

**Issue by SilentGlasses** [midnightprioriem/mkdocs-autolinks-plugin#18]
Was just wondering if there is a way to add an exclude list to not try to link to files that would allow me to add this as an example of how to use the plugin:
```
[Page Name](page_name.md)
![Image](image.png)
```
but not get these warnings:

```
WARNING -  AutoLinksPlugin unable to find page_name.md in directory /git_repo/docs
WARNING -  AutoLinksPlugin unable to find image.png in directory /git_repo/docs
```

Was thinking in `mkdocs.yml` you can set something like this:

```yml
plugins:
  - search
  - autolinks:
      exclude:
        - page_name.md
        - image.png
```

**Comment by Rwarcards762** [comment]
If the goal is to try and "show how to use the plugin" would wrapping the items in backticks not be sufficient, ie:

`[Page Name](page_name.md)`

or

```md
[Page Name](page_name.md)
```

> I haven't had the chance to use the plugin yet, but would hope that it wouldn't autofill into items wrapped in backticks...

**Comment by midnightprioriem** [comment]
Yeah, the ideal solution would be to fix #1.

**Comment by SilentGlasses** [comment]
Solution is as @midnightprioriem said so closing this out.
